db/controller.js

```diff
@@ -0,0 +1,13 @@
+const endpoints = require('../endpoints.json');
+const { fetchTopics } = require('./model')
+
+exports.apiHealthCheck = (req, res) => {
+    res.status(200).send({ endpoints })
+}
+
+exports.getTopics = (req, res) => {
+    fetchTopics()
+        .then((topics) => {
+            res.status(200).send({ topics })
+        })
```

Comment:
there is no error handling required for this endpoint but this needs a catch block so any errors from the promise chain are still caught.